Formula/apache-arrow-glib.rb

```diff
@@ -0,0 +1,39 @@
+class ApacheArrowGlib < Formula
+  desc "GObject Introspection files of Apache Arrow"
+  homepage "https://arrow.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.7.1/apache-arrow-0.7.1.tar.gz"
+  sha256 "f8f114d427a8702791c18a26bdcc9df2a274b8388e08d2d8c73dd09dc08e888e"
+  head "https://github.com/apache/arrow.git"
+
+  depends_on "apache-arrow"
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gobject-introspection"
+
+  def install
+    cd "c_glib" do
+      args = %W[
+        --prefix=#{prefix}
+        CC=#{ENV.cc}
+      ]
+      system "./configure", *args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    ENV["ARCHFLAGS"] = ""
```

Comment:
Why?

Reply:
I heard from @kou that `-arch i386` is supplied by Python's distutils without setting empty`ARCHFLAGS`, and then the build was failed.
So I specify the empty `ARCHFLAGS` here.

Reply:
I don't have High Sierra environment, so I've never confirmed that by myself.
It is just in case.

Reply:
We filter that out automatically.